### ISSUE TITLE
Will you release the code of 3D classification on PeRFception dataset?

### ISSUE THREAD

**Issue by imlixinyang** [POSTECH-CVLab/PeRFception#13]

**Comment by jeongyw12382** [comment]
Yes. We are currently cleaning the code. We'll let you know right after we release the code.

**Comment by imlixinyang** [comment]
Hi @jeongyw12382. I really appreciate your contribution on this work and want to know if there is an estimated date on the release plan.

**Comment by hugoycj** [comment]
@imlixinyang Thanks for the great work! I am also wondering when will the downstream code and checkpoints will be released.

**Comment by jeongyw12382** [comment]
Sorry for being late. We had some problems on uploading codes for downstream tasks.
Since many people requested codes for downstream tasks, we have pre-released the code. 

We have uploaded the code for downstream on the repository below
- https://github.com/POSTECH-CVLab/NeRF-Downstream

I want to remind all that this code is a pre-released version and we will enhance its docs and codes ASAP. 
So, I recommend to just simply refer some implementation details from the repository. 
I'll let you know if we finish our refacotring.
